### PR TITLE
First release

### PR DESCRIPTION
**Pull request by jillesca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,102 @@
-# cisco-nso-interactive-quick-start
-Cisco NSO interactive quick start
+# Cisco NSO interactive Quick Start
+
+Explore and experiment the world of NSO on Cisco Code Exchange, where you can dive right in and try out examples in a user-friendly browser-based integrated development environment (IDE).
+
+But wait, there's more! You can also contribute your own examples and share them with everyone. The best part is, your peers can play with your examples without having to install anything.
```
Comment (cohult) [inline README.md]:
```suggestion
But wait, there's more! You can also contribute your examples and share them with everyone. The best part is your peers can play with your examples without installing anything.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,102 @@
...
+
+## How can I add my own example?
+
+1. Simply, submit your repo on <https://developer.cisco.com/codeexchange/submit/>
```
Comment (cohult) [inline README.md]:
```suggestion
Simply submit your repo on <https://developer.cisco.com/codeexchange/submit/>
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,102 @@
...
+2. Once your submission is approved, your repository will be featured on the [Cisco Cloud IDE](https://developer.cisco.com/codeexchange/search/?complexity=devenv)
+3. That's it! Now you can start exploring, playing, and sharing your awesome examples. Let the creativity flow!
+
+## Need an example?
+
+This short example will demostrate how you can setup a simulated network of Cisco-ios routers and how to manage these with NCS in Code Exchange Cloud IDE. NCS will talk Cisco CLI towards the routers.
```
Comment (cohult) [inline README.md]:
```suggestion
This short example will demonstrate how to set up a simulated network of Cisco IOS routers and manage these with NSO in Code Exchange Cloud IDE. NSO will talk Cisco CLI towards the routers.
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,102 @@
-# cisco-nso-interactive-quick-start
-Cisco NSO interactive quick start
+# Cisco NSO interactive Quick Start
+
+Explore and experiment the world of NSO on Cisco Code Exchange, where you can dive right in and try out examples in a user-friendly browser-based integrated development environment (IDE).
```
Comment (cohult) [inline README.md]:
```suggestion
Explore and experiment with the world of NSO on Cisco Code Exchange, where you can dive right in and try out examples in a user-friendly browser-based integrated development environment (IDE).
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,102 @@
...
+ncs --version
+ncs --status | grep -i status
+```
+
+Configure NSO
+
+```bash
+ncs_cli -C -u admin
+devices sync-from
+devices device ios1 check-sync
```
Comment (cohult) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,102 @@
...
+config
+show full-configuration devices device ios1 config | nomore
+devices device ios0 config
+ios:hostname nso.cisco.com
+top
+commit dry-run outformat native
+commit
+```
+
+### Explore and play with NSO Example Collection
```
Comment (cohult) [inline README.md]:
```suggestion
### Explore and play with the NSO Example Collection
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,102 @@
...
+
+Go to `~/nso-6.1.2.1` > `examples.ncs` in VS Code workspace or use the terminal
```
Comment (cohult) [inline README.md]:
```suggestion
Go to `$NCS_DIR` > `examples.ncs` in the VS Code workspace or use the terminal
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,102 @@
...
+
+- Code shared is public, **avoid any confidential information.**
+
+## Recommendations
+
+- It is recommended to avoid specifying NSO versions in the code, as the underlying NSO will be upgraded with newer releases.
+
+## FAQ
+
+- What NSO install type is using?
```
Comment (cohult) [inline README.md]:
```suggestion
- What NSO install type is used?
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,102 @@
...
+  - NSO local install. But System install is in the pipeline.
+- What NEDs come with the container?
+  - Example NEDs. These should be considered as examples. Any NEDs present in the container is only in order to run the simulated networks. Do not use these for real devices.
```
Comment (cohult) [inline README.md]:
```suggestion
  - Example NEDs. These should be considered as examples. NEDs in the container are only used to run the simulated networks. Do not use these with real devices.
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,102 @@
...
+    cisco-iosxr-cli-3.0
+    cisco-iosxr-cli-3.5
+    cisco-nx-cli-3.0
+    dell-ftos-cli-3.0
+    juniper-junos-nc-3.0
+    developer:~ > ls -1 $NCS_DIR/packages/neds/ | wc -l
+    10
+    developer:~ >
+    ```
+- My code is approved but I don't see it?
```
Comment (cohult) [inline README.md]:
```suggestion
- My code is approved, but I don't see it?
```

**Review by cohult — CHANGES_REQUESTED** [review]

File: README.md [hunk 11]
```diff
@@ -1,2 +1,102 @@
-# cisco-nso-interactive-quick-start
-Cisco NSO interactive quick start
+# Cisco NSO interactive Quick Start
```
Comment (cohult) [inline README.md]:
```suggestion
# Cisco NSO Interactive Quick Start
```

**Review by cohult — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 2 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 3 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 4 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 5 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 6 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 7 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 8 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 9 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 10 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

File: README.md (hunk 11 above)
Comment (jillesca) [reply README.md]: Fixed - 5fae8cf7cda98b119122133f2908f5b91c73c08c

**Review by cohult — APPROVED** [review]

**Pull request merged**